### DISCUSSION TITLE
Compile LAMMPS libraries (liblamps_mpi.a, liblammps_serial.a) with MACE

### DISCUSSION THREAD

**Discussion by fcannizzaro-byte** [ACEsuit/mace#532] [Q&A]
I successfully build LAMMPS with MACE for CPU using the instructions from here: https://mace-docs.readthedocs.io/en/latest/guide/lammps.html. 

I need to compile these two libraries: liblamps_mpi.a, liblammps_serial.a.  I have been looking into ./src/Makefile but could not find any option there. As reference, in the case of GAP using QUIP, I would do this:

```
make clean-all; make yes-manybody no-python yes-REAXFF yes-ml-quip serial mode=lib
make clean-all; make yes-manybody no-python yes-REAXFF yes-ml-quip  mpi    mode=lib
```

When compiling LAMMPS libraries, is there an option to add MACE?

**Comment by wcwitt** [comment]
Apologies, I don't fully understand the question. But you might start by reviewing the LAMMPS documentation on building with CMake. For example: https://docs.lammps.org/Build_basics.html#build-the-lammps-executable-and-library.
